docs/jenkins.md

```diff
@@ -54,35 +54,23 @@ https://github.com/theforeman/foreman-infra/tree/master/puppet/modules contains
 
 ### Configuring a new node
 
-For Enterprise Linux 8:
-
-* Ensure yum.puppet.com is configured: [puppet7-release](https://yum.puppet.com/puppet7-release-el-8.noarch.rpm)
-* `yum -y install puppet-agent`
-* `echo "server = puppet.theforeman.org" >> /etc/puppetlabs/puppet/puppet.conf`
-* ensure hostname is set node0X.jenkins.<provider\>.theforeman.org where <provider\> is osuosl or aws for example and that the record is in DNS
-* Make the `puppet` command available: `source /etc/profile.d/puppet-agent.sh`
-* `puppet ssl bootstrap`
-* Sign the certificate on the puppetmaster or via Foreman
-* `puppet agent --test`
-* Set the host group to "Builders" in Foreman
-* Run `puppet agent --test` twice (second run is important, due to the rvm module behaviour)
+For Enterprise Linux 9, ensure the correct Puppet repository is configured:
 
+* Ensure yum.puppet.com is configured: [puppet7-release](https://yum.puppet.com/puppet7-release-el-9.noarch.rpm)
+* `dnf -y install https://yum.puppet.com/puppet7-release-el-9.noarch.rpm`
 
-For Enterprise Linux 7:
+For Enterprise Linux:
 
-* Ensure EPEL is configured: [epel-release](https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm)
```

Comment:
Are you sure we no longer need EPEL? Pretty sure we had some dependencies that we relied on. Though I think we should configure EPEL via Puppet if we need it, rather than making it something to bootstrap from.

Reply:
Nothing failed and I did not configure this.